metagraph/src/graph/alignment/alignment.cpp

```diff
@@ -489,11 +871,10 @@ Json::Value Alignment::path_json(size_t node_size, std::string_view label) const
 }
 
 Json::Value Alignment::to_json(std::string_view full_query,
-                               const DeBruijnGraph &graph,
+                               size_t node_size,
```

Comment:
Isn't node_size a property of the alignment, so it should be stored there?

Reply:
It's a property of the graph to which the alignment was done. If we stick to the assumption that the alignment is done against a fixed-order de Bruijn graph, then node_size can be inferred from the stored sequence, the path, and the alignment's first-node offset.